include/aspect/initial_conditions/S40RTS_perturbation.h

```diff
@@ -31,7 +31,7 @@ namespace aspect
   {
     using namespace dealii;
 
-    namespace internal
+    namespace S40RTSinternal
```

Comment:
You can't just use the names `S40RTS` because that's already the name of the class.

Just go with
```
namespace internal
{
  namespace S40RTS
  {
    class SphericalHarmonics;
```
This is the style we use everywhere else already.